bin/export-record-barcodes

```diff
@@ -63,6 +63,10 @@ BARCODE_FIELDS = [
     "scan_id",
     "scan_id_kiosk",
     "scan_id_manual",
+
+    # SCAN: Tiny Swabs
+    "ans_collection_id",
+    "tiny_collection_id",
```

Comment:
One interesting thing - when I run `make` to build the database locally, I received logging errors `OSError: [Errno 40] Message too long` because the debug logs were turned on and the message was too long when the redcap `fetch` tries to log its parameters with `LOG.debug(f"Requesting content={content} from REDCap with params {loggable_parameters} for {self}")` - did you see that as well? Perhaps the additional two variables here pushed it over the edge for message length. Not specifically an issue with the PR per se, but it exposes that we might want to do something upstream in `id3c.cli.redcap` to be a little more elegant and split up the logging.

Reply:
Yeah, I got the same thing. I've seen this on other commands as well, so handling it upstream makes sense.